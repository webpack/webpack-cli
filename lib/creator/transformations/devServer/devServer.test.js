"use strict";

const defineTest = require("../../../transformations/defineTest");

defineTest(__dirname, "devServer", "devServer-0",{
	contentBase: "path.join(__dirname, 'dist')",
	compress: true,
	port: 9000
});
defineTest(__dirname, "devServer", "devServer-1", "someVar");
