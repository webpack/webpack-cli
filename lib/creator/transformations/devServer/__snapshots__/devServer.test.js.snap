// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`devServer transforms correctly using "devServer-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    devServer: {
        contentBase: path.join(__dirname, 'dist'),
        compress: true,
        port: 9000
    }
}
"
`;

exports[`devServer transforms correctly using "devServer-1" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    devServer: someVar
}
"
`;
